README.md

```diff
@@ -1,19 +1,31 @@
-# Stereum 2.0
+![GitHub - Stereum 2 0 - ReadMe](https://user-images.githubusercontent.com/82385103/175423093-03d28d7b-3c2e-44b7-8be1-7f2a5946fcc6.png)
+
+---
 
 [![Support Server](https://img.shields.io/badge/Discord-Stereum-blue?style=flat-square)](https://discord.gg/8Znj8K6GjN) [![Support Email](https://img.shields.io/badge/Email-Stereum-brightgreen?style=flat-square)](mailto:stereum@stereum.net)
 
-This repository contains the code and scripts for ~~[Stereum 2.0](https://stereum.net/ethereum-node/)~~ (tba), an [Ethereum](https://ethereum.org/) node setup and management software with focus on self sovereignity, privacy and flexibility.
+---
+This repository contains the code and scripts for Stereum 2.0 (tba)
+
+
+🍄 Stereum is a tool to manage the process of setting up & maintaining an Ethereum node for you with a heavy focus on self sovereignty, privacy and flexibility. Stereum is actively developed by the RockLogic GmbH, who first started to develop on its predecessor in 2019.
+
+🍄 Stereum 2.0 aims to be the most flexible way to leverage your Ethereum node for staking, data science, development or your own personal use case. We hope to explore every hermit’s dream with you!
 
-## Get Started
+🍄 With Stereum you are able to participate in the Ethereum network and its various protocols (like SSV)!
+
+
+## Get Started 
 Pre-built software packages are available on our [website](https://stereum.net/) or on GitHub releases. After installing the launcher start it up and connect to a server of your choice. Take a look at the ~~[guide](https://stereum.net/ethereum-node/)~~ (tba) to find out what server operating systems are supported and what you can do with Stereum!
```

Comment:
This is incorrect, the packages are available on the release page of this repo. Something like `Downloads of the Stereum Launcher are available down below`, because we got the download section right below. Or just leave this sentence out altogether?

Reply:
The sentence got the boot 😄